c10/core/DispatchKey.h

```diff
@@ -241,6 +241,10 @@ enum class DispatchKey : uint16_t {
 
   ZeroTensor, // registered at build/aten/src/ATen/RegisterZeroTensor.cpp
 
+  // Out-of-core key for Fake Tensor in torchdistx.
+  // See https://pytorch.org/torchdistx/latest/fake_tensor.html
+  Fake,
```

Comment:
Are you sure that you want `Fake` to have higher priority than ZeroTensor/Negative/Conjugate? (I think I'd expect it to have ~ the same priority as the `Python` key, right above backend select, or you might hit weird issues with code that uses forward mode AD / complex numbers)